oot.sh

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/bash
+
+while getopts ':c:' OPT; do
+        case ${OPT} in
+        c)
+		case ${OPTARG} in
+		clang|gcc|linaro_gcc)
```

Comment:
How about `if [ -f "$_self_dir/compiler_$OPTARG.sh" ]`, using `_self_dir` from below?

Reply:
Here we're only checking the option name and unfortunately we can only do pattern matching.

Reply:
Huh, what am I missing? Let me set up a suggestion to show what I mean.